### ISSUE TITLE
BEDLAM: A Synthetic Dataset of Bodies Exhibiting Detailed Lifelike Animated Motion

### ISSUE THREAD

**Issue by Arata2887** [Arata2887/issue_copy#25]
1.どんな論文? /What does this work do?

- https://arxiv.org/abs/2306.16940
- CVPR 2023 (Highlight Paper)
- Proposes BEDLAM, a synthetic video dataset contains monocular RGB videos together with ground truth 3D bodies in SMPL-X format.
- By trained on BEDLAM, even basic methods like HMR achieve SOTA performance(Cliff) on real images. 

2.先行研究と比べてどこがすごい? /What makes this work greater than existing works?

<img width="875" alt="8a2cd63215ba1dd92ba2e4106f8be68" src="https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/eea102bd-2e73-4a60-a252-2e767d5d60cc">

- First time show that neural networks trained only on synthetic data achieve state-of-the-art accuracy on the problem of **3D human pose and shape (HPS)** estimation from real images. 
- Achieving **sufficient realism**, unlike Previous synthetic datasets, which have been small, unrealistic, or lacked realistic clothing. 

3.技術や手法のキモはどこ? /What is the heart of this techno- logy or method?

Each step in the process of creating BEDLAM is explained below and illustrated in **Fig. 2**.

<img width="878" alt="38bacf237215a4fe05026845988a8a4" src="https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/d9532b14-d70e-4d9e-ab25-d385dff175fa">

**Body shapes:** Sample body shapes from a diverse pool of 271 body shapes in total.The ground truth body shapes are represented with 11 shape components in the SMPL-X gender-neutral shape space.
**Skin tone diversity:** Use 50 female and 50 male commercial skin albedo textures from Meshcapade with minimal clothing and a resolution of 4096x4096. These artistcreated textures represent a total of seven ethnic groups with multiple variations within each. few examples are shown in **Fig. 3**.

> <img width="425" alt="02ff3f85cef54ee1c9008953e0eb15c" src="https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/0c7b6b4a-fc3b-4561-8d2a-21b2556be282">

**3D Clothing and textures:** hired a 3D clothing designer to create 111 unique real-world outfits, and design 5 to 27 clothing textures with different colors and patterns using WowPatterns. In total have 1691 unique clothing textures (see **Fig. 4)**.
For high-BMI bodies, auto-transfer the textures of 1738 simulated garments onto the body UV-map using Blender. Then render high-BMI body shapes using these textures (see **Fig. 5**).

> <img width="416" alt="1bbbf5c692f1f5b4770ad82de9d662d" src="https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/f1e8637a-c149-4250-8a71-93b0fd871c56">
> <img width="433" alt="afaf40bd90623b7ca8edc9dcbe940ba" src="https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/99829934-0213-4189-b694-c910903f8386">

**Hair:** Use the Character Creator (CC) software from Reallusion and purchased hairstyles to generate 27 hairstyles (**Fig. 6**).

> <img width="425" alt="4ed2698785b453c3eb44701ab0d3aa9" src="https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/f2f7784d-fac6-4f44-99e9-7420e9473513">

**Summary:** BEDLAM is generated from a combination of 271 bodies, 27 hairstyles, 111 types of clothing, with 1691 clothing textures, 2311 human motions, in 95 HDRI scenes and 8 3D scenes, with on average 1-10 person per scene, and a variety of camera poses. See **Sup. Mat.** for detailed statistics.

4.どうやって有効だと検証した? /How is this work validated?

Train both **HMR** and **CLIFF** on the synthetic data (BEDLAM+AGORA) using an HRNet-W48 backbone and refer to these as **BEDLAM-HMR** and **BEDLAM-CLIFF** respectively.
**Quantitative results: Table 1** summarizes the key results. 

> <img width="426" alt="5f9a228a28c800546a8974380ad00ba" src="https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/90835fd5-453f-43b2-ade4-c79618ed78df">
> 
> ①Pre-training on BEDLAM and fine-tuning with a mix of 3DPW and BEDLAM training data **gives the most accurate results on 3DPW and RICH** (i.e. BEDLAM-CLIFF* is more accurate than CLIFF†* ). 
> ②Using the same training, makes HMR (i.e. BEDLAM-HMR*) nearly as accurate on 3DPW and more accurate than CLIFF†* on RICH. This **suggests that even simple methods can do well if trained on good data**.

**Qualitative results:** All these benchmarks are shown in **Fig. 7**. Although authors do not assign gender labels to any of the training data, on test data, methods trained on BEDLAM predict appropriately gendered body shapes. That is, BEDLAM have automatically learned the association between image features and gendered body shape.

> <img width="877" alt="51e0c42e2544b685290ffc7081a6970" src="https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/35f30380-20d1-4c61-9cba-6b6b13169f55">
**Ablation studies: Table 3** shows the effect of varying datasets, backbone weights and percentage of data; see **Sup. Mat**.

> <img width="426" alt="b91c5e49607de596763bf9bad17f0df" src="https://github.com/jr-tagawalab-nttcom/paper-survey/assets/119403000/2ec149f2-7c5c-4844-902f-48e53479747c">

5.議論はある? /Any discussion for this work?

6.読んでいてわからなかったところは? /What don't you understand for this paper?

7.公開コードやデータセットは? /Public codes or datasets?

https://github.com/pixelite1201/BEDLAM

8.次に読むべき論文は? /What should we read next
